src/svg/flash.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" width="51.29" height="128.38" viewBox="0 0 51.29 128.38">
```

Comment:
`51.29`, `128.38` These are odd numbers ^^
Can you please resize the svg with round numbers? Also, we usually put svg with a width (or height) that doesn't exceed `32px`